### ISSUE TITLE
Example Search controls

### ISSUE THREAD

**Issue by elodou01** [flamencist/ldap4net#40]
**Describe the bug**
I am trying to use the LdapConnection and PageResultRequestControl that you use in your example which is supposed to support ldap v3 search controls, but the class PageResultRequestControl doesn't exist. Neither does the Controls attribute.

**To Reproduce**
```
var searchRequest = new SearchRequest(baseDN, filter, scope, attributes)
                    {
                        AttributesOnly = false,
                        TimeLimit = TimeSpan.Zero,
                        Controls =
                        {
                            new PageResultRequestControl(sizeLimit)
                            {
                                IsCritical = true,
                            },
                        },
                    }
```

**Expected behavior**
Controls and PageResultRequestControl are supposed to exist without the DirectoryServices package.

**Desktop :**
 - OS: Windows 10 Family
 - Library version : 05/22/2020 version cloned
 - .NET\core\mono version : 
 - LDAP server : OpenLdap

**Comment by flamencist** [comment]
It's implemented but not deployed yet. Will be available in 2.5 version. Please wait.

**Comment by flamencist** [comment]
released
